docs/upgrading/graphcommerce-6-1-to-6-2.md

```diff
@@ -87,3 +87,67 @@ In your package.json
     "@graphql-mesh/json-schema": "latest",
     "@graphql-mesh/transform-filter-schema": "latest",
 ```
+
+## Upgrading your Hygraph schema
+
+In version 6.2 and onwards it is possible to upgrade your Hygraph schema using
+the Graphcommerce Hygraph CLI. This is a tool that automatically modifes your
+Hygraph schema without the need of adding models/components/enumarations
+manually. The following steps are needed to utilize this tool:
+
+1. Install the @graphcommerce/hygraph-cli package
+
+2. Create a Hygraph migration token
+
+   1. Open your Hygraph project. Go to: Project settings > Permanent auth tokens
+
+   2. Click 'Add token' and give it a name, something like 'GraphCommerce Write
+      Access Token' and keep stage on 'Published'.
+
+   3. Under 'Management API', click 'Yes, Initialize defaults'
+
+   4. Click 'Edit Permissions' and enable: 'Update' and 'Delete' permissions for
+      'models', 'enumerations', 'fields', 'components' and 'sources':
+
+      - Update existing models
+
+      - Delete existing models
+
+      - Update existing fields
+
+      - Delete existing fields
+
+      - Update existing enumerations
+
+      - Delete existing enumerations
+
+      - Update existing components
+
+      - Delete existing components
+
+      - Update remote sources
+
+      - Delete remote sources
+
+      - Read existing environments
+
+      - Read public content views
+
+      - Create public content views
+
+      - Update public content views
+
+      - Delete public content views
+
+      - Can see schema view
```

Comment:
This seems a very long list. Is there also an 'admin' token that has access to everything or is the user required to select from the list? Might be a suggestion that a user either can create a token with these actions or an admin token.

Also mention that the token is _only_ required during a migration and nowhere else.